src/app/view/toggle-switch-button/toggle-switch-button.component.html

```diff
@@ -1,7 +1,11 @@
-<div class="toggle-switch-button-component">
-  <span *ngIf="!!labelFalse" [ngClass]="createLabelCheckedTag(false)" (click)="onToggle(false)">{{
-    labelFalse
-  }}</span>
+<div class="toggle-switch-button-component" [ngClass]="{disabled: disabled}">
+  <span
+    *ngIf="labelFalse"
+    [ngClass]="createLabelCheckedTag(false)"
+    (click)="!disabled && onToggle(false)"
```

Comment:
Nit: would be a little bit less verbose to check for `disabled` in the `onToggle` function, instead of each time it's called.